packages/core/src/SkipMenu/SkipMenu.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from '../Flex'
+import { Link } from '../Link'
+import { SrOnly } from '../SrOnly'
+
+const OffScreenPanel = styled(SrOnly)`
+  &:focus-within {
```

Comment:
This css attribute is not supported by IE11, so if we are planning on supporting that browser we will need to figure out an alternative.

Reply:
Maybe just add a comment denoting this for future reference